docs/_includes/date-picker.html

```diff
@@ -0,0 +1 @@
+<!-- TODO: add this before merging; HTML structure is very large so it probably shouldn't be on the main skin site -->
```

Comment:
We need to add the structure for review. I can't review the CSS until I know the HTML it relates to.

We might even want to think about adding a "calendar" module, which would be the most granular level. For example, a "calendar" could be just the data-table part, which is then used inside of a "date picker" along with other UI such as title, button forward, button back and all day buttons. Reason for this modularity is that there might be other components that require a calendar (a static, read-only calendar display for example).

Reply:
For now, the HTML structure is demonstrated in the storybook only because it's very large. I discussed with Andrew offline-- for now, we're thinking that it will just be in the storybook.

Reply:
Yes i agree for now. Lets just keep it in storybook. 
I had the same concern when I first looked but storybook helped a lot with what the structure should be
We can show a portion of it but imo its not adding much. It would have a large amount of text in the example for each date.

Reply:
I would actually prefer we add the structure here too. It doesn't have to be anything extensive; the basic one-month view would be good enough. Since this is the first stop for devs to get more info on implementation or reference DOM/CSS, we should really have something here.

Also, it's very odd to have a "Date Picker" link in the sidebar that doesn't work, as is the case now.

Reply:
Agree with Arthur. Please create the necessary documentation on the website. We cannot just rely on storybook here. We don't have to add multiple months, or even multiple dates, but the website is the first port of call for understanding our modules.

Reply:
Also please bear in mind what I said about modularity/flexibility. Is the underlying calendar HTML/BEM versatile enough to handle a calendar of static dates (i.e not buttons in table cells) as well as an interactive calendar (i.e. buttons in table cells)?

Reply:
Resolving this as we do now have example code on the web site.